SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/WorkingSetSystem.cpp

```diff
@@ -266,8 +294,23 @@ TSet<Worker_EntityId_Key> FWorkingSetSystem::FImpl::DeclineConflictingRequests(c
 		Worker_EntityId MarkerEntityId;
 		TSet<Worker_EntityId_Key> ConflictingEntities;
 
-		static bool Compare(const FConflictingWorkingSetData& Lhs, const FConflictingWorkingSetData& Rhs)
+		static bool Compare(const FConflictingWorkingSetData& Lhs, const FConflictingWorkingSetData& Rhs, const FWorkingSetState& LhsState,
+							const FWorkingSetState& RhsState, FEntityToWorkerMap MapEntityToOwningWorker)
 		{
+			// Special case: prioritize sets with all entities in a set are owned by the same worker.
```

Comment:
Suggestion comments aren't working, but -
`// Special case: prioritize sets _when_ all entities in a set are owned by the same worker.`